### ISSUE TITLE
Error running command sshfs.add, can't connect

### ISSUE THREAD

**Issue by TheBlyatMan** [SchoofsKelvin/vscode-sshfs#288]
The extension worked for a day, then today I decided to log on to complete some lab exercises and it wouldn't connect, and gives me this message in the bottom right:

Error running command sshfs.add: Running the contributed command: 'sshfs.add' failed.. This is likely caused by the extension that contributes sshfs.add.

The output files for SSH FS just says "Connecting to <server address>" and shows nothing else when it fails.

**Comment by SpanishPear** [comment]
This happened to me when I had the wrong host. (ie i had `login@blah.blah.blah` instead of `login.blah.blah.blah`)

**Comment by SchoofsKelvin** [comment]
This issue would be easier to solve with debug logs, so please follow these steps:
- Add `DEBUG_SSH2` to the `sshfs.flags` array in VS Code's User Settings (settings.json)
    e.g. `"sshfs.flags": ["DEBUG_SSH2"]`
- If you already have a connection open, close it completely (or even reload the window)
- Go to `Output > SSH FS` and copy the log from there after replicating your bug
-  While it should censor passwords/passphrases, I recommend checking it for (other) sensitive data first.
    _(especially since `DEBUG_SSH2` activates some internal logging, which is less likely to be censored)_

**Comment by TheBlyatMan** [comment]
Shrey????????????????? Dayum bro, didn't expect to see you here, almost thought it wasn't you but your dyed hair gave you away. Anyways the problem just seems to pass and go, and the last time I got it, I fixed it by connecting to the uni vpn. Super weird but the problem vanished today. Oh and about the debug logs, it would just get stuck and "Connecting to <server address>" and doesn't update when the error message pops up.
